frontend/src/metabase/models/containers/ModelDetailPage/ModelDetailPage.tsx

```diff
@@ -111,7 +111,9 @@ function ModelDetailPage({
     const card = model.card();
     const isModel = model.isDataset();
     if (isModel) {
-      loadMetadataForCard(card);
+      if (model.database()) {
```

Comment:
@kulyk does it make sense for you?

Reply:
I think it does, is it needed to handle a scenario when we're missing data permissions for the model's database? If so, I think it might make sense to patch on the `loadMetadataForCard` action level, WDYT? Shouldn't block that PR though

Reply:
> is it needed to handle a scenario when we're missing data permissions for the model's database

maybe, I don't know. I'm afraid we'll need to pass a `model` or `database` to `loadMetadataForCard` in this case and I'm not sure how simple can be extension of the current method as it's used in several places and definitely relied on card mostly